src/snoowrap.js

```diff
@@ -233,6 +233,87 @@ const snoowrap = class snoowrap {
       return requester;
     });
   }
+
+  static get grantType () {
+    return {
+      CLIENT_CREDENTIALS: 'client_credentials',
+      INSTALLED_CLIENT: 'https://oauth.reddit.com/grants/installed_client'
+    };
+  }
+  /**
+  * @summary Creates a snoowrap requester from a "user-less" Authorization token
+  * @desc In some cases, 3rd party app clients may wish to make API requests without a user context. App clients can request
+  * a "user-less" Authorization token via either the standard client_credentials grant, or the reddit specific
+  * extension to this grant, https://oauth.reddit.com/grants/installed_client. Which grant type an app uses depends on
+  * the app-type and its use case.
+  * @param {object} options
+  * @param {string} options.userAgent A unique description of what your app does. This argument is not necessary when snoowrap
+  is running in a browser.
+  * @param {string} options.clientId The client ID of your app (assigned by reddit). If your code is running clientside in a
+  browser, using an "Installed" app type is recommended.
+  * @param {string} [options.clientSecret] The client secret of your app. Only required for "Client" grant type.
+  * @param {string} [options.deviceId] A unique, per-device ID generated by your client. Only required
+  * for "Installed" grant type.
```

Comment:
It might also be worth mentioning the string `DO_NOT_TRACK_THIS_DEVICE` here, which reddit recommends using if a device shouldn't be tracked. (This could also be nice when using app-only oauth in a browser, because it's difficult to get a consistent device ID.)